src/pages/angels.js

```diff
@@ -1,11 +1,17 @@
-import React, { useState, useEffect } from "react"
+import React from "react"
 import BrandLayout from "../components/BrandLayout"
 import SEO from "../components/seo"
 import { Button } from "@material-ui/core"
 
 const AngelsPage = () => {
-  const stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
-  const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")
+  const STRIPE_KEY = "pk_test_XiirziufElakjoOpyuyCrPfo"
```

Comment:
This is not technically wrong, but we usually define them right under the imports.